docs/start/hello-world.ipynb

```diff
@@ -7,23 +7,64 @@
    "source": [
```

Comment:
<p>"To save some time, you can instead use the following code to run this small workload on in the Qiskit Runtime local testing mode to use..."</p><p> </p><p>typo "on in"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>